sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -44,7 +50,11 @@ import org.apache.spark.sql.catalyst.plans.logical.{LogicalPlan, Project}
  *                 unless they are specified with full qualified table name with database prefix.
  */
 case class CreateViewCommand(
-    tableDesc: CatalogTable,
+    name: TableIdentifier,
+    userSpecifiedColumns: Seq[(String, Option[String])],
+    comment: Option[String],
+    properties: Map[String, String],
+    originalText: Option[String],
     child: LogicalPlan,
     allowExisting: Boolean,
     replace: Boolean,
```

Comment:
Shall we replace `allowExisting` and `replace` with `SaveMode`? We have the following mapping here:

| `allowExisting` | `replace` |    `SaveMode`   |
|:---------------:|:---------:|:---------------:|
|      `true`     |  `false`  |     `Ignore`    |
|     `false`     |  `false`  | `ErrorIfExists` |
|     `false`     |   `true`  |   `Overwrite`   |
|      `true`     |   `true`  |   `Overwrite`   |

`Append` can't be used here though.